ppxlib.opam

```diff
@@ -23,7 +23,7 @@ depends: [
   "stdlib-shims"
   "ocamlfind"               {with-test}
   "re"                      {with-test & >= "1.9.0"}
-  "cinaps"                  {with-test & >= "v0.12.1"}
+  "cinaps"                  {with-test & >= "v0.12.1" & ocaml >= "4.07"}
```

Comment:
This leads to `false` in all cases (the `ocaml` variable is not defined)

Reply:
ah interesting, so is there an actual way to express that?